src/geo/map/plain-layer.js

```diff
@@ -11,6 +14,28 @@ var PlainLayer = LayerModelBase.extend({
     className: 'plain',
     color: '#FFFFFF',
     image: ''
+  },
+
+  initialize: function (attrs, options) {
+    attrs = attrs || {};
+    options = options || {};
+    if (!options.vis) throw new Error('vis is required');
+
+    this._vis = options.vis;
+
+    this.bind('change', this._onAttributeChanged, this);
```

Comment:
We are using `listenTo` in the new Builder code, and I feel it would be great if we make use of the same convention here (only for Backbone objects). How does it sound @alonsogarciapablo @donflopez?